src/main/java/rx/internal/operators/TakeLastQueueProducer.java

```diff
@@ -36,9 +36,7 @@ public TakeLastQueueProducer(NotificationLite<T> n, Deque<Object> q, Subscriber<
         this.subscriber = subscriber;
     }
 
-    private volatile long requested = 0;
-    @SuppressWarnings("rawtypes")
-    private static final AtomicLongFieldUpdater<TakeLastQueueProducer> REQUESTED_UPDATER = AtomicLongFieldUpdater.newUpdater(TakeLastQueueProducer.class, "requested");
+    private AtomicLong requested = new AtomicLong();
```

Comment:
Extending `AtomicLong` exposes the inherited methods outside of this class. Do you think it's worth the confusion that that might bring?